robottelo/cli/base.py

```diff
@@ -112,8 +138,8 @@ def create(cls, options=None):
             cls._construct_command(options), output_format='csv')
 
         # Extract new object ID if it was successfully created
-        if len(result.stdout) > 0 and 'id' in result.stdout[0]:
-            obj_id = result.stdout[0]['id']
+        if len(result) > 0 and 'id' in result[0]:
```

Comment:
no update needed. i've just always preferred the version i mentioned. something we can mull around though.